dali/test/python/checkpointing/test_dali_checkpointing.py

```diff
@@ -1203,7 +1193,7 @@ def pipe(arg):
         glob="The pipeline does not support checkpointing*"
         "because it contains operator*outside the pipeline*",
     ):
-        p.build()
```

Comment:
I'd recommend keeping the `build` here so we can pinpoint the source of the error better.